classes/02class/exercises/c02-network09/starkmatt/c02-network09.txt

```diff
@@ -8,3 +8,11 @@ I then clicked Inbound Rules on the Security Group and added SSH from my home IP
 
 - For the private-sg, I used the same methodology as above but with the diffence was giving SSH access only from my public subnet
 
+Inbound
+
+SSH	TCP	22	0.0.0.0/0	SSH from Public Subnet
```

Comment:
SSH from everywhere in the internet? Is that safe?

Reply:
Technically not at all!
But in this case, the private subnet is only accessible via the public subnet through the NAT Gateway.
Would this make it OK ?
Or is putting the public IP from the jumpbox a better idea ?

Reply:
would be safer to add the VPC subnet in that rule instead of 0.0.0.0/0? Another option, maybe add the SG that is attached to the jumpserver? In that case, only machines with that SG attached to it would be able to connect to a resource with that SG?
There are many options better than the 0.0.0.0/0.

Reply:
Roger that!
Understood I'll make the changes !